tests/nexus/guest/src/main.rs

```diff
@@ -0,0 +1,18 @@
+#![cfg_attr(target_arch = "riscv32", no_std, no_main)]
+
+use nexus_rt::println;
+
+#[nexus_rt::main]
+#[nexus_rt::private_input(x)]
+fn main(x: u32) {
```

Comment:
I think the input has to be `Vec<u8>` because we already serialize it outside of Nexus SDK, and we need to handle the deserialization from the result of `InputItem::as_bytes`.